### PR TITLE
Update tests for latest SDK and fix bug in validator return types

### PR DESCRIPTION
**Pull request by BrianBlum**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by BrianBlum** [comment]
@tjprescott, will you be reviewing this PR?

File: src/command_modules/azure-cli-ams/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.4.6
++++++
+* Bug fix for validator return types.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-ams/HISTORY.rst]: This should be phrased in a way that describes what customer-facing behavior is fixed. Customers don't know what a validator is.

Comment (BrianBlum) [reply src/command_modules/azure-cli-ams/HISTORY.rst]: Changed the comment to be more specific about the behavior

File: src/command_modules/azure-cli-ams/HISTORY.rst [hunk 2]
```diff
@@ -5,7 +5,7 @@ Release History
 
 0.4.6
 +++++
-* Bug fix for validator return types.
+* Return ISO 8601 time strings for archive-window-length and key-frame-interval-duration.
 * Tests updated for new SDK requirements.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-ams/HISTORY.rst]: Tests are not customer facing and should not be mentioned

**Comment by BrianBlum** [comment]
@tjprescott Can you please merge these changes. Thanks.

**Pull request merged**